tests/ci/clickhouse_helper.py

```diff
@@ -32,7 +32,13 @@ def _insert_json_str_info_impl(url, auth, db, table, json_str):
         }
 
         for i in range(5):
```

Comment:
but we still do `raise InsertException(error)` on invalid response or 5 exceptions

Reply:
That's correct, it's caught below.